src/charm.py

```diff
@@ -37,8 +37,8 @@
 CERTIFICATE_COMMON_NAME = "nssf.sdcore"
 
 
-class NSSFOperatorCharm(CharmBase):
-    """Main class to describe juju event handling for the SD-Core NSSF operator."""
+class NSSFK8sOperatorCharm(CharmBase):
```

Comment:
I think there's no need to change the class name.

Reply:
It kept as same.